README.md

```diff
@@ -151,6 +151,27 @@ The script allows the user to specify variables, their units, plotting
 style (stacked vs overlaid), as well as the format of the output image.
 Use `-h` for more information.
 
+
+### Data Compression
+
+The `prmon_compress_output.py` script (Python3) can be used to compress the output file
+while keeping the most relevant information.
+The compression algorithm works as follows, for each separate series: 
+* For rapidly changing metrics: for any three points A, B, and C, if the linear
+interpolation between A and C passes by B ± *threshold* * *range(metric)*, then B is deleted.
+The percentage threshold is set by the `--precision` parameter.
+* For more steady metrics: only keep the changepoints.
```

Comment:
Reading again, `rapidly changing` and `steady` metrics might be confusing. Looking at the script, the latter essentially contains the counts of processes, threads, and gpus while the former everything else. Maybe it's better to spell them out.

Reply:
Yes, naming is definitely off.

Would `MEMORY_IO_NETWORK_GPU_USAGE` and `NPROCS_NTHREADS_NGPUS` work better for you?

Reply:
(I thought you were quoting the code)

I will fix this in the README for sure. I will also change the variable names in the code to avoid confusion.